chipy_org/apps/meetings/serializers.py

```diff
@@ -4,7 +4,7 @@
 
 
 class PresentorSerializer(serializers.ModelSerializer):
-    email = serializers.SerializerMethodField('get_email')
```

Comment:
My understanding of Rest Framework is that this won't work after removing what method to call?